lib/CSPUpdater.js

```diff
@@ -0,0 +1,50 @@
+
+var jsdom = require('jsdom');
+var fs = require('fs');
+var Q = require('q');
+var path = require('path');
+
+// This snippet grants all permissions necessary for the app to access the BrowserSync Server and thus enables us to do LiveReload
+var CSP_SNIPPET = "\n\n\n<!-- BEGIN -- Added by cordova-plugin-livereload--> \n<meta http-equiv=\"Content-Security-Policy\" content=\"default-src *; style-src * 'unsafe-inline' 'unsafe-eval'; script-src 'self' 'unsafe-inline' 'unsafe-eval' *;\"> \n <!-- END -- Added by cordova-plugin-livereload -->\n\n\n";
```

Comment:
This is really long to read >> 80 characters - can we split it up? Github PR review in particular, makes this harder to review.